enigma/src/main/resources/lang/en_us.json

```diff
@@ -28,6 +28,7 @@
 	"menu.file.stats": "Mapping Stats...",
 	"menu.file.stats.title": "Mapping Stats",
 	"menu.file.stats.top_level_package": "Top-level package:",
+	"menu.file.stats.synthetic_parameters": "Include synthetic parameters",
```

Comment:
Same here, use title case (while you're at it, change the one for top-level package too)